velox/functions/sparksql/tests/JsonObjectKeysTest.cpp

```diff
@@ -41,6 +41,14 @@ TEST_F(JsonObjectKeysTest, basic) {
   assertEqualVectors(jsonObjectKeys(R"(1)"), expected);
   assertEqualVectors(jsonObjectKeys(R"("hello")"), expected);
   assertEqualVectors(jsonObjectKeys(R"("")"), expected);
+  assertEqualVectors(
+      jsonObjectKeys(R"({"key": 45, "random_string"})"), expected);
+  assertEqualVectors(jsonObjectKeys(R"([{1, 2, 3}])"), expected);
+  // test with UNCLOSED_STRING error of simdjson
```

Comment:
Nit: comment style. Capitalize initial letter and end up with period.

Reply:
Sure, updated